backend/src/helpers/UserHelper.js

```diff
@@ -0,0 +1,42 @@
+// Priorities are set from the lowest to the highest.
+const statePriorities = [ 'away', 'viewing', 'commenting', 'editing', 'merging' ];
+
+class UserHelper {
+	static getUserListFromSessions( issueSessions ) {
+		const users = {};
+		for ( const socketId in issueSessions ) {
+			if ( !users[ issueSessions[ socketId ].id ] ) {
```

Comment:
Assign `issueSessions[ socketId ]` to some variable

Reply:
Sure.